app/models/user.rb

```diff
@@ -99,8 +99,7 @@ def original_login_token
   def regenerate_reset_password_token
     @original_reset_password_token = SecureRandom.urlsafe_base64
     update!(
-      reset_password_token:
-        Digest::SHA2.base64digest(@original_reset_password_token)
+      reset_password_token: Digest::SHA2.base64digest(@original_reset_password_token)
```

Comment:
Incorrect formatting, please use the [recommended config](https://docs.pupilfirst.com/developers/development_setup#code-formatting)